### PR TITLE
Fix a few minor editorial issues.

### PR DESCRIPTION
**Pull request by msporny**
This PR clarifies the purpose of registries (one word change), removes a double "of of", and fixes the dates on the expected timeline.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -146,15 +146,15 @@ <h2>Scope</h2>
         </p>
         <ul>
           <li>Data models for verifiable credentials and presentations</li>
-          <li>Registries for the data models</li>
+          <li>Registries for extending the data models</li>
```
Comment (brentzundel) [inline index.html]:
```suggestion
          <li>Registries for the data models</li>
```

iirc during the WG meeting, folks wanted to make sure we could have registries that weren't explicitly about extending the data model.

Comment (msporny) [reply index.html]: Hrm, I'm trying to think of a "data model" registry that would fall into that category. I can't think of one, but that's probably lack of imagination on my end.

I'm happy to put it back to the way it was.

Comment (brentzundel) [reply index.html]: Cool.
I don't recall the specifics, but my notes for requested changes by the group led me to remove the "extensions" language in [3369ef3](https://github.com/w3c/vc-wg-charter/commit/3369ef3ec08db7b7f42748fd581625a338fd0dd9) just last week :)

File: index.html [hunk 2]
```diff
@@ -356,20 +356,55 @@ <h2>Coordination</h2>
...
-            <dt><a href="">Decentralized Identifier Working Group</a></dt>
-            <dd><i class="todo">[specific nature of liaison]</i></dd>
-          </dl>
+            <h3 id="w3c-coordination">W3C Groups</h3>
+            <dl>
+                <dt><a href="https://www.w3.org/RCH/WG/">RDF Canonicalization and Hashing Working Group</a></dt>
+                <dd>
+                    To synchronize on canonicalization output expression mechanisms that might be used by the Linked Data Integrity specification.
+                </dd>
+
```
Comment (iherman) [inline index.html:365]: This must be VC Linked Data Integrity spec, in sync with the list of deliverables.

**Review by iherman — APPROVED** [review]

File: index.html [hunk 3]
```diff
@@ -356,20 +356,55 @@ <h2>Coordination</h2>
...
-          <h3 id="w3c-coordination">W3C Groups</h3>
-          <dl>
-            <dt><a href="">Decentralized Identifier Working Group</a></dt>
-            <dd><i class="todo">[specific nature of liaison]</i></dd>
-          </dl>
+            <h3 id="w3c-coordination">W3C Groups</h3>
+            <dl>
+                <dt><a href="https://www.w3.org/RCH/WG/">RDF Canonicalization and Hashing Working Group</a></dt>
+                <dd>
+                    To synchronize on canonicalization output expression mechanisms that might be used by the Linked Data Integrity specification.
```
Comment (brentzundel) [inline index.html]: per Ivan's suggestion:
```suggestion
                    To synchronize on canonicalization output expression mechanisms that might be used by the VC Linked Data Integrity specification.
```

**Review by brentzundel — APPROVED** [review]

File: index.html (hunk 2 above)
Comment (msporny) [reply index.html:365]: Applied changes suggested by @iherman and @brentzundel -- all changes are in, IIUC, this PR is ready to merge.

**Pull request merged**